.github/workflows/post-merge-deploy-to-dev.yml

```diff
@@ -0,0 +1,74 @@
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+name: Deploy to Amazon ECS
+
+on:
+  push:
+    branches:
+      - main1
```

Comment:
If you want to test this out before merging it, you could use the same approach I'm using over in stubs - https://github.com/alphagov/di-ipv-stubs/blob/468d02c79451a97e1753d366e9aac2a69d5a2427/.github/workflows/core-stub-image.yml#L7 - it's a temporary branch that'll always invoke this GHA, and once we're  merged, then we can delete it.

It looks like it might be a bit of a painful workflow with pushing on main and then branching to main`1` to see it deploy.